src/main/cli/cli.c

```diff
@@ -1036,6 +1035,144 @@ static const char *processChannelRangeArgs(const char *ptr, channelRange_t *rang
     return ptr;
 }
 
+/**
+ * @brief Converts parsing error into a message and prints it.
+ */
+static void intParsePrintError(
+    enum intParseError_e err,
+    long int fromVal,
+    long int toVal,
+    const char *cmdName,
+    const char *argName
+)
+{
+    switch (err) {
+    case INT_PARSE_ERROR_NOT_IN_RANGE:
+        cliShowArgumentRangeError(cmdName, (char *)argName, fromVal, toVal - 1);
+        break;
+    case INT_PARSE_ERROR_NOT_A_NUMBER:
+        cliShowArgumentNotANumberError(cmdName, argName);
+        break;
+    case INT_PARSE_ERROR_END_OF_LINE:
+        cliPrintErrorLinef(cmdName, "INVALID ARGUMENT COUNT");
+        break;
+    }
+}
+
+/**
+ * @brief Parses positive integer argument from a command line
+ * @param cmdline is a double pointer to a string to parse. It will be updated
+ *        if successfully parsed.
+ * @param cmdName is a command name. Used in error messages
+ * @param argName is an argument name. Used in error messages
+ * @retval Parsed value on success or negative on failure
+ */
+static long int processPositiveIntArg(
+    const char * *const cmdline,
```

Comment:
Please check https://betaflight.com/docs/development/CodingStyle#spaces
```suggestion
    const char **const cmdline,
```